pandas/core/series.py

```diff
@@ -5745,7 +5745,49 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0,
+        doc="""
+        The index (axis labels) of the Series.
+        Get the index (row labels) of the Series.
+
+        Returns
+        -------
+        Index
+            The index labels of the Series.
+
+        See Also
+        --------
+            Series.reindex:
+                Conform Series to new index.
+            Series.set_index:
+                Set Series as DataFrame index.
+            Index:
+                The base pandas index type.
```

Comment:
Looking at other examples in the code base, e.g. [this](https://github.com/pandas-dev/pandas/blob/9271d25ca4cfa458deb068ccf14d2654516ff48a/pandas/io/parsers/readers.py#L425-L429), this should probably be

```suggestion
        See Also
        --------
        Series.reindex : Conform Series to new index.
        Series.set_index : Set Series as DataFrame index.
        Index : The base pandas index type.
```